subprojects/docs/src/docs/userguide/gradleWrapper.adoc

```diff
@@ -15,106 +15,204 @@
 [[gradle_wrapper]]
 == The Gradle Wrapper
 
-Most tools require installation on your computer before you can use them. If the installation is easy, you may think that’s fine. But it can be an unnecessary burden on the users of the build. Equally importantly, will the user install the right version of the tool for the build? What if they’re building an old version of the software?
+The recommended way to execute any Gradle build is with the help of the Gradle Wrapper (in short just “Wrapper”). The Wrapper is a feature that makes is extremely easy to execute a Gradle project without a Gradle installation. Upon invocation, it can _automatically download, unpack and install the Gradle runtime with a predefined version_ and use it to execute the build. As a result, developers can get up and running with a Gradle project quickly without having to follow manual installation processes saving your company time and money.
```

Comment:
I would prefer

> The Wrapper is a script that invokes a declared version of Gradle, downloading it beforehand if necessary.

to 

> The Wrapper is a feature that makes is extremely easy to execute a Gradle project without a Gradle installation.

because it explains what the wrapper _really is_.

Reply:
I am fine with that.